bobtimus/src/http.rs

```diff
@@ -59,14 +59,16 @@ where
         warp::reject::reject()
     })?;
 
-    bobtimus
-        .handle_create_swap(payload)
-        .await
-        .map(|message1| warp::reply::json(&message1.transaction))
```

Comment:
Yeah I don't know why we ever serialized this a JSON here? I've changed this to the hex-representation here: https://github.com/comit-network/droplet/pull/74

Reply:
I saw your change! Is bincode okay, or should we stick to hex? I first considered hex but I gave up after trying to deserialise it into a `Transaction` and failing.

Reply:
I would prefer hex as it seems the _natural_ way to serialize a transaction. Happy to have a look at the deserializing problem together.